MonoGame.Framework/Platform/Utilities/SignalWatcher.cs

```diff
@@ -0,0 +1,53 @@
+using System;
+using System.Runtime.InteropServices;
+
+namespace MonoGame.Utilities
+{
+    public static class SignalWatcher
```

Comment:
Might want to make this private/internal?

Reply:
Ups, yes.